### PR TITLE
adjust boxplot with updated denominator

### PR DESCRIPTION
**Pull request by elizabethmcd**
updated code for boxplot figure, adjusted denominator to reflect the total number of filtered proteins in A. americanum and not total proteins directly from gene prediction, since all other tick species reference comparisons are to the filtered amount

### CODE REVIEW

File: validation/scripts/validate-plot-protein-lengths.R [hunk 1]
```diff
@@ -46,40 +40,34 @@ diamond_proportion_plot <- diamond_tables_clean %>%
...
   count() %>% 
-  mutate(protein_hits = n) %>% 
-  select(-n)
+  mutate(proportion_of_amblyomma = (n / 28319) * 100) # divided by total proteins in A. americanum for more than 2 hits
+
 
 tick_species_metadata_hits <- left_join(tick_species_protein_hit_counts, tick_species_metadata) %>% 
   mutate(proportion_of_reference = protein_hits / total_protein_count) %>% 
-  mutate(proportion_of_amblyomma_query = protein_hits / 34557) %>% 
+  mutate(proportion_of_amblyomma_query = protein_hits / 28319) %>%
```
Comment (taylorreiter) [inline validation/scripts/validate-plot-protein-lengths.R:66]: do you want to code these as numbers instead of hard coding so they auto update? no worries if not since this is a one off script

File: validation/scripts/validate-plot-protein-lengths.R [hunk 2]
```diff
@@ -46,40 +40,34 @@ diamond_proportion_plot <- diamond_tables_clean %>%
...
-
-# combine with tick metadata for total protein counts
-tick_species_protein_hit_counts <- diamond_tables_clean %>% 
+  filter(n > 2) %>% 
   group_by(species_name) %>% 
   count() %>% 
-  mutate(protein_hits = n) %>% 
-  select(-n)
+  mutate(proportion_of_amblyomma = (n / 28319) * 100) # divided by total proteins in A. americanum for more than 2 hits
+
```
Comment (taylorreiter) [inline validation/scripts/validate-plot-protein-lengths.R:62]: this diff is hard to follow -- you don't need to filter to below .9 this time since ...? or is it doing something different than before? (also it looks like this isn't saved just printed out so probs fine no matter what

**Review by taylorreiter — APPROVED** [review]
lgtm! couple questions, totally understand if it doesn't make sense to make changes

File: validation/scripts/validate-plot-protein-lengths.R (hunk 1 above)
Comment (elizabethmcd) [reply validation/scripts/validate-plot-protein-lengths.R:66]: I think just hard code because it's a one-off thing

File: validation/scripts/validate-plot-protein-lengths.R (hunk 2 above)
Comment (elizabethmcd) [reply validation/scripts/validate-plot-protein-lengths.R:62]: Interesting, I think before I wanted some stats and was playing around with what number of proteins had length proportions greater than 90% of the reference protein, and I would change that number to see different distributions. I must have reuused that part to get this more than 2 protein hits within a reference. But yes was just printing to see different subsets of results

Comment (elizabethmcd) [reply validation/scripts/validate-plot-protein-lengths.R:62]: I had quite a few of these non-saved lines of code and just printing to see how things looked and I cleaned that up, so I think that's what happened here but I wanted to keep the filtering for more than 2 hits since that final check became important
